sysroot/usr/include/kernel/lib/semaphore.hpp

```diff
@@ -8,13 +8,17 @@
  * @copyright Copyright the Panix Contributors (c) 2020
  * 
  */
+#ifndef PANIX_SEMAPHORE_HPP
+#define PANIX_SEMAPHORE_HPP
```

Comment:
Why not use `#pragma once`?

Reply:
See #174 and its associated MR. Resolved in 67712a8.